waddrmgr/address.go

```diff
@@ -62,11 +65,11 @@ type ManagedPubKeyAddress interface {
 	ManagedAddress
 
 	// PubKey returns the public key associated with the address.
-	PubKey() *btcec.PublicKey
+	PubKey() (*btcec.PublicKey, error)
 
 	// ExportPubKey returns the public key associated with the address
 	// serialized as a hex encoded string.
-	ExportPubKey() string
+	ExportPubKey() (string, error)
```

Comment:
This may not be needed anymore with the PubKey method changes, since the caller would be able to create the exported key in compressed or uncompressed serialized formats themselves.